specs/recover_safe/01_main.rst

```diff
@@ -0,0 +1,183 @@
+==========================================================
+Recover Safe
+==========================================================
+
+=====  ============  =========  ==========
+epic      title       author     created
+=====  ============  =========  ==========
+`90`_  Recover_Safe  tschubotz  2019-03-28
+=====  ============  =========  ==========
+
+.. _90: https://github.com/gnosis/safe/issues/90
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+-------------------------------
+
+- Users are switching to a new phone and want to continue using their old Safe.
+- Users deleted the Gnosis Safe app and would like to continue using their old Safe.
+
+Inputs
+-----------
+
+- Safe address
```

Comment:
I think it would be good not to request Safe address. As I understand this spec is for describing the current solution. Should we also create an epic with recovery improvement where we ask recovery phrase only and our service can suggest safe(s) for selection?

Reply:
yea for a future solution it would be great to not have the user enter the address. I would tackle that inside the recovery option epic.